docs/guides/_include/pool-creation.rst

```diff
@@ -51,3 +53,7 @@ Create the zpool
       * ``keyformat=passphrase`` - By setting the format to ``passphrase``, we can now force a prompt for this in
         ``zfsbootmenu``. It's critical that your passphrase be something you can type on your keyboard, since you will
         need to type it in to unlock the pool on boot.
+
+.. note::
+
+  Pools are created with the feature flags supported by the current ZFSBootMenu binary (EFI) release. If your system uses a newer version of OpenZFS and you plan on building from source, ``-o compatibility=openzfs-2.1-linux`` can be omitted.
```

Comment:
What if this is part of the above note?

```suggestion

      * ``compatibility=openzfs-2.1-linux`` fixes the feature flags to those supported by the current ZFSBootMenu binary (EFI) release. This will prevent an errant ``zpool upgrade`` from rendering your system unbootable. If you use a newer OpenZFS and intend to keep custom ZFSBootMenu images sychronized with the host, you can omit this option.
```

Reply:
Also, if this is adopted, maybe change "However, the following options need to be addressed" on line 46 above to "However, the following options merit special attention"

Reply:
The above note section only applies to encrypted pools. Moving it below the tab group ensures that it's shown for both encrypted and un-encrypted pools.

Reply:
This is why I get for trying to garner context from a patch. In that case I guess I'd revise the wording to:
```suggestion

.. note::

  The option ``-o compatibility=openzfs-2.1-linux`` ensures that the pool is created only with feature flags supported by the current ZFSBootMenu binary release. If you plan on building a custom ZFSBootMenu image that you will keep synchronized with your host, the compatibility option may be omitted.
```